docs/connector-development/config-based/connector-builder-ui.md

```diff
@@ -1,79 +1,78 @@
 # Connector Builder UI
 
-The connector builder UI provides an ergonomic iteration interface on top of the [low-code YAML format](understanding-the-yaml-file/yaml-overview). We recommend using it to iterate on your YAML connectors. 
+The connector builder UI provides an ergonomic iteration interface on top of the [low-code YAML format](understanding-the-yaml-file/yaml-overview). We recommend using it to iterate on your low-code connectors.
 
-:::caution 
-The connector builder UI is in alpha, which means it’s still in active development and may include backward-incompatible changes. Share feedback and requests with us on our Slack channel or email us at feedback@airbyte.io 
+:::caution
+The connector builder UI is in alpha, which means it’s still in active development and may include backward-incompatible changes. Share feedback and requests with us on our Slack channel or email us at feedback@airbyte.io
 :::
 
 ## Getting started
-The high level flow for using the connector builder is as follows: 
+
+The high level flow for using the connector builder is as follows:
 
 1. Run the Airbyte webapp to access the connector builder
-2. Use the connector builder to iterate on your YAML (either starting with a new YAML, or pasting an existing one into the editor)
+2. Use the connector builder to iterate on your low-code connector
 3. Export the YAML into a low-code connector module on your local machine
 4. Build the connector's Docker image
 5. Use the built connector image in Airbyte
 
 ### Run an Airbyte instance
+
 :::info
-You need at least Airbyte v0.40.19 to run the connector builder. 
+You need at least Airbyte v0.40.27 to run the connector builder UI.
 :::
 
 :::tip
-We recommend using a dedicated Airbyte instance for the connector builder UI. This will allow you to pick up connector builder UI upgrades without impacting your production data pipelines. 
+We recommend using a dedicated Airbyte instance for the connector builder UI. This will allow you to pick up connector builder UI upgrades without impacting your production data pipelines.
 :::
 
 The connector builder UI is bundled as part of the Airbyte webapp. To run it, follow [the instructions for deploying Airbyte locally](../../deploying-airbyte/local-deployment) to run Airbyte on your machine. Once you do, Airbyte should be reachable at `http://localhost:8000`.
 
-
 ### Visit the connector builder
-Once your Airbyte instance has started and you've moved past the initial setup screen, visit `http://localhost:8000/connector-builder`. You will be redirected to a URL of the form `localhost:8000/workspaces/<UUID>/connector-builder` where `<UUID>` is the ID automatically generated for your workspace by Airbyte. On this page you will find the connector builder UI. It should look like this: 
```

Comment:
Removed this because it no longer redirects when visiting the connector builder page